frame/elections-phragmen/src/lib.rs

```diff
@@ -267,6 +277,28 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Maximum number of votes per voter that allowed by the election data provider.
+		#[pallet::constant]
+		type MaxVotesPerVoter: Get<u32>;
+
```

Comment:
```suggestion
```

This crate already has a constant value for this (16), and it does not need to be configurable.